pephub/routers/api/v1/namespace.py

```diff
@@ -67,9 +67,7 @@ async def get_namespace(
 
         namespace: databio
     """
-    nspace = nspace.model_dump()
-    nspace["projects_endpoint"] = f"{str(request.url)[:-1]}/projects"
```

Comment:
Were we using this for anything?

Reply:
Not really, I am testing and can't find any issue